### ISSUE TITLE
Corona Mtl - Support

### ISSUE THREAD

**Issue by gopikrishnachaganti** [BabylonJS/Exporters#490]
Hi, when 3dsmax has the Corona renderer, the max scene will have Corona Materials generated. these assets are not properly going thru Babylon GLTF exporter as the exporter is ignoring the Corona materials along with textures while exporting gltf. 

i knew Arnold, VRay & Mental ray support is there with Babylon exporters. Can you please help on Corona renderer while exporting gltf?

**Comment by deltakosh** [comment]
Hey! Do you want to contribute? If you have C# skill it could be easy to do. We are already handling a lot of requests so even though this one is interesting it could take a while before being able to add it

**Comment by Drigax** [comment]
I agree, this looks relatively straightforward, but I have few other feature requests to prioritize before I can get to this. I'll toss this on my personal backlog, but if you think you have an implementation, I'd be more than happy to help get it into the source quickly. 😊

**Comment by gopikrishnachaganti** [comment]
@Drigax @deltakosh i am also in similar boat with my backlog, will inform you if i can build this up.

**Comment by Remi-Tribia** [comment]
Hi, it would greatly improve our workflow if this was implemented.

I may be able to help if someone points me in the right direction (what files/classes should I dive into for a starting point).

**Comment by pandaGaume** [comment]
Hello @Remi-Tribia, I know this is an old issue but if you still would like to know more about the direction to go for implement this, let me know.
G.

**Comment by Remi-Tribia** [comment]
> Hello @Remi-Tribia, I know this is an old issue but if you still would like to know more about the direction to go for implement this, let me know.
> G.

Sure I'm still interested!

**Comment by pandaGaume** [comment]
@Remi-Tribia this is great. Have you already setting up your dev environement ?

**Comment by Remi-Tribia** [comment]
@pandaGaume Not yet, I have VS Community 2019 & Rider installed though.

**Comment by pandaGaume** [comment]
Great, VS community is everything you need to use with Git. Get the repo and try it with max. Once you will be able to compile, i wil give you direction about where the Material code is located, and how you may add the corona one.

**Comment by Priya12592** [comment]
Hi @pandaGaume I'm looking to export GTLF from my 3DS with Corona renderer and it throws an error. It would be great if you can help me with this please.
![Error](https://user-images.githubusercontent.com/87110998/124874217-cf786200-e00a-11eb-8de2-f7f71fa8e954.JPG)

**Comment by pandaGaume** [comment]
@Priya12592 as Remi sayed, the Corona Material is still unsupported.
G.

**Comment by Priya12592** [comment]
Thanks @pandaGaume @Remi-Tribia. Appreciate your response.

Cheers!
